cli/schedule-get.go

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 The Gort Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/getgort/gort/client"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	scheduleGetUse   = "get"
+	scheduleGetShort = "Get a list of schedules commands"
+	scheduleGetLong  = "Get a list of schedules commands."
+	scheduleGetUsage = `Usage:
+gort schedule get [flags]
+
+Flags:
+  -h, --help  Show this message and exit`
+)
+
+func GetScheduleGetCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   scheduleGetUse,
+		Short: scheduleGetShort,
+		Long:  scheduleGetLong,
+		RunE:  scheduleGetCmd,
+	}
+
+	cmd.SetUsageTemplate(scheduleGetUsage)
+
+	return cmd
+}
+
+func scheduleGetCmd(*cobra.Command, []string) error {
+	c, err := client.Connect(FlagGortProfile, FlagConfigBaseDir)
+	if err != nil {
+		return err
+	}
+
+	info, err := c.SchedulesGet()
+	if err != nil {
+		return err
+	}
+
+	maxId, maxAdapter, maxChannel, maxCron := 2, 7, 7, 4
```

Comment:
Should these be constants?
Also, can you comment them to make it clearer what they do and maybe why they're the values they are?

Reply:
Definitely not constants, I use them to properly space my table.